client/views/msg.tpl

```diff
@@ -1,5 +1,5 @@
 <div class="msg {{type}}{{#if self}} self{{/if}}{{#if highlight}} highlight{{/if}}" id="msg-{{id}}" data-time="{{time}}"{{#if from.nick}} data-from="{{from.nick}}"{{/if}}>
-	<span class="time tooltipped tooltipped-e" aria-label="{{localetime time}}">
+	<span class="time tooltipped tooltipped-e tooltipped-no-delay" aria-label="{{localetime time}}">
```

Comment:
Yeah so this was particularly horrible without delay, because since times are like a vertical barrier between chat and channel list, you would *always* trigger them...

Reply:
fair